lib/foreman/dynflow.rb

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+require 'dynflow'
+
+module Foreman
```

Comment:
Yes, in my opinion Foreman should just define the hook points that change (like `action_logger`, `dynflow_logger`). Everything else looks like it should be the same on all Rails applications, including the daemon.rb and most of configuration.rb